Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings

```diff
@@ -578,13 +578,13 @@
 "customSearchEngine.deleteEngineAlertTitle" = "%@ arama motorunu silmek istediğinizden emin misiniz?";
 
 /* The warning description shown to user when custom search engine already exists. */
-"customSearchEngine.engineAlertExistsAlertDescription" = "Aynı adda bir arama motoru zaten var.";
+"customSearchEngine.engineAlertExistsAlertDescription" = "Aynı isimde bir arama motoru zaten var.";
 
 /* A description explaining that search template url is insecure, it is used for instance - Insecure Custom Search Template for Brave Search, Brave Search is a search engineand on a new seperate line */
-"customSearchEngine.insecureSearchTemplateURLErrorDescription" = "Şunun için Güvenli Olmayan Özel Arama Şablonu:";
+"customSearchEngine.insecureSearchTemplateURLErrorDescription" = "Güvenli Olmayan Özel Arama Şablonu:\" is enough no need for \"Şunun için";
```

Comment:
oh shoot my bad, this should be 

"customSearchEngine.insecureSearchTemplateURLErrorDescription" = "Güvenli Olmayan Özel Arama Şablonu:";